openfasoc/generators/temp-sense-gen/tools/readparamgen.py

```diff
@@ -5,7 +5,6 @@
 import os
 import re
 import sys
-import time
```

Comment:
Why is this removed?

Reply:
The library was not used so I removed the import statement.